src/server/dancelor_server.ml

```diff
@@ -108,9 +111,7 @@ let initialise_logs () =
 
 let populate_caches () =
   Dancelor_server_controller.Version.Svg.populate_cache ();%lwt
-  Dancelor_server_controller.Version.Pdf.populate_cache ();%lwt
```

Comment:
I'm slightly surprised that the version pdf cache is also removed, but I might be missing something

Reply:
Because I said “Now also done for version PDFs.”!

Reply:
But I guess this warrants renaming the PR.